components/atoms/FilterExperiments.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "next-i18next";
+
+/**
+ *  Filter Experiments component
+ */
+export function FilterExperiments(props) {
+  const { t } = useTranslation("common");
+  const createOptions = props.options.map((option) => (
+    <div role="radio" key={option.id}>
+      <input
+        type="radio"
+        className="radioButton absolute -left-full -top-full"
+        name="radioButtonFilter"
+        defaultChecked={option.id === 0}
+        value={option.text}
+        id={option.id}
+        onChange={props.onChange}
+      />
+      <label
+        className={`float-left text-xs px-5 py-3 cursor-pointer border border-solid border-gray-600 border-opacity-50 mr-0 mb-6 ${
```

Comment:
Don't forget to use "font-body" class to make sure your labels are using the "Noto Sans" font.

Reply:
It has been done.